pwiz_tools/Skyline/Controls/Graphs/DetectionsToolbar.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms;
+using pwiz.Skyline.Model;
+using pwiz.Skyline.Util;
+using Settings = pwiz.Skyline.Controls.Graphs.DetectionsGraphController.Settings;
+using IntLabeledValue = pwiz.Skyline.Controls.Graphs.DetectionsGraphController.IntLabeledValue;
+
+namespace pwiz.Skyline.Controls.Graphs
+{
+    partial class DetectionsToolbar :  GraphSummaryToolbar        //UserControl 
+    {
+        private Timer _timer;
+        private static Bitmap _emptyBitmap = new Bitmap(1, 1);
+
+        private Dictionary<ToolStripDropDown, ToolStripItem> _selectedItems =
+            new Dictionary<ToolStripDropDown, ToolStripItem>(4);
+
+        public override bool Visible => true;
+
+        public DetectionsToolbar(GraphSummary graphSummary) : base(graphSummary)
+        {
+            InitializeComponent();
+            _timer = new Timer { Interval = 100 };

```

Comment:
This Timer might need to get Dispose'd.

Reply:
I do not see any memory leaks, but I guess it's better to be on the safe side.